tests/transforms/augmentation/test_random_gamma.py

```diff
@@ -0,0 +1,31 @@
+from torchio import RandomGamma
+from ...utils import TorchioTestCase
+from numpy.testing import assert_array_equal
+
+
+class TestRandomGamma(TorchioTestCase):
+    """Tests for `RandomGamma`."""
+    def test_with_zero_gamma(self):
+        transform = RandomGamma(log_gamma_range=0)
+        transformed = transform(self.sample)
+        assert_array_equal(self.sample.t1.data, transformed.t1.data)
```

Comment:
If an operation will be performed, we should use `assert_array_almost_equal`. Otherwise, we can detect that the input is `0` and do nothing (see #292).